manim/mobject/mobject.py

```diff
@@ -1731,6 +1731,29 @@ def get_boundary_point(self, direction):
         index = np.argmax(np.dot(all_points, np.array(direction).T))
         return all_points[index]
 
+    def get_midpoint(self) -> np.ndarray:
+        """Get coordinates of the middle of the line that forms the  :class:`~.Mobject`
```

Comment:
I don't really like 'Line' here. Why not path ?

Reply:
Agreed, it shouldn't be specific to `Line`, especially since this is currently implemented in Mobject.